distributions/openhab/pom.xml

```
@@ -139,6 +139,7 @@
                         <feature>service</feature>
                         <feature>system</feature>
                         <feature>openhab-runtime-base</feature>
+                        <feature>openhab-ui-default</feature>
```

I'd like to keep the `openhab-ui-` prefix for real add-ons.
Let's consider the UI to be a central part for runtime and call it `openhab-runtime-ui`, which is a good fit to the `base` feature.
```suggestion
                        <feature>openhab-runtime-ui</feature>
```